jobs/github.rb

```diff
@@ -1,4 +1,5 @@
 
+require 'yaml'
```

Comment:
If launching job separately, outside of dashing - failed with unrecongnized constants (YAML)